index.html

```diff
@@ -30,7 +30,7 @@
                     <a href="#">Contact</a>
```

Comment:
- [ ] The page title is poor, please consider using a better page like: `Mark's portfolio`. this will make the title describe the content of the page